tcms/tests/__init__.py

```diff
@@ -227,3 +227,26 @@ def setUpTestData(cls):
         cls.execution_4 = executions[3]
         cls.execution_5 = executions[4]
         cls.execution_6 = executions[5]
+
+
+class AssignPermissionsToTestUser(LoggedInTestCase):
+    """Assign required permissions to user """
+
+    def setUpTestData(cls):
+        super(LoggedInTestCase, cls).setUpTestData()
+
+        cls.tester = LoggedInTestCase.tester
```

Comment:
1. Think will not work as you expect it to. It should be `cls.tester = super().tester`
2. you don't need this, because your child class gets all the instance properties from the inherited. `tester` as such gets assigned to `AssignPermissionsToTestUser` without you having to do anythin
3. having this in mind, you don't actually need to define `setUpTestData` on this class at all, because all it is doing is calling the parent method, but this happens anyway, so just omit this method
```python
class LoggedInTestCase:
    def setUpTestData():
        print("Setting up test data")

class AssignPermissionsToTestUser(LoggedInTestCase):
    pass

perm = AssignPermissionsToTestUser()
perm.setUpTestData()
```
run this snippet in a shell or a file and you will see first-handedly what I mean

Reply:
> Think will not work as you expect it to. It should be cls.tester = super().tester

Say again ?

Reply:
My point was that `LoggedInTestCase.tester` would throw an error, because my assumption was that`tester` is an instance member, not a class one. However, my assumption was wrong, and `tester` is indeed a class member. So this code will work, and my point 1 is invalid.